package.json

```diff
@@ -46,5 +46,8 @@
   },
   "peerDependencies": {
     "webpack": "^1.13.1"
+  },
+ "publishConfig": {
+    "registry": "https://artifactory.dkandu.me/artifactory/api/npm/coursera-npm-packages"
```

Comment:
i remember why i didn't have this here before — i didnt want to publicize this endpoint (this  is a public repo). we dont _need_ this here.

Reply:
I see. removing.